spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/core/AwsClientBuilderConfigurer.java

```diff
@@ -76,4 +86,24 @@ public Region resolveRegion(@Nullable AwsClientProperties clientProperties) {
 				? Region.of(clientProperties.getRegion())
 				: this.regionProvider.getRegion();
 	}
+
+	public static @Nullable MetricPublisher createSpecificMetricPublisher(MetricPublisher metricPublisher,
+			AwsClientProperties properties) {
+		if (ClassUtils.isPresent("software.amazon.awssdk.metrics.publishers.cloudwatch.CloudWatchMetricPublisher", null)
```

Comment:
`ClassUtils.isPresent("software.amazon.awssdk.metrics.publishers.cloudwatch.CloudWatchMetricPublisher", null)` should be stored in a static field so that we make sure that we call it once